example/play_with_ovs/play_with_ovs.go

```diff
@@ -41,25 +44,38 @@ func play(ovs *libovsdb.OvsdbClient) {
 }
 
 func createBridge(ovs *libovsdb.OvsdbClient, bridgeName string) {
+	api := ovs.Apis["Open_vSwitch"]
```

Comment:
Thanks, looks neat. 
strings like Open_vSwitch, Bridge, etc can be moved to a common constant as its being used repetitively at many places.

Reply:
Good idea, as a nice by-product of using constants, it lets users distinguish between "Open_vSwitch" the Database and "Open_vSwitch" the table. :)